amis/build_ami.sh

```diff
@@ -20,9 +20,10 @@
 #   build-date: timestamp to append to the AMIs names (optional)
 
 requirements_check() {
-    packer build --help >/dev/null 2>&1
-    if [ $? -ne 0 ] ; then
-      echo "packer command not found. Is Packer installed?"
+    currentver="$(packer --version)"
```

Comment:
We used "packer build" command because there is another "packer" tool out there, probably accepting the "--version" option

Reply:
Good point. Let me check `packer build --version`

Reply:
Ok, I've updated to do two checks:
1. Check `packer build` (this is to distinguish from the default cracklib packer binary present on RHEL based AMI's)
2. Check `packer --version` >= 1.4.0.